### ISSUE TITLE
How to identify the candidate trigger on the original training data ?

### ISSUE THREAD

**Issue by linmou** [thunlp/Adv-ED#13]
According to part 3.4 of the paper, authors used the small-scale labeled data to train the encoder and discriminator.  But what should we do if there is no extra data?  Do we need to identify the candidate triggers on dev and test set also?
By the way, since the NYT corpus is really huge, how many sentences did you use for augment data when semi-supervised training ?

**Comment by linmou** [comment]
Particularly, in the paper, DMBERT without Boot also performed pretty good. So I want to reproduce DMBERT with out extra data also.

**Comment by Bakser** [comment]
Hi, thanks for your interest in our work.
1. The method introduced in Section 3.4 is for semi-supervised and distant supervision settings. If there is no extra unlabeled data, the process cannot be performed. And the process in Section 3.4 is not used for dev and test sets.
2. I cannot recall the exact number. The trigger-based latent instance discovery process is performed on the whole NYT corpus and this filters most of the sentences out.
3. If you only want to run DMBERT in the supervised setting, I will refer to the codes for our recent work (https://github.com/THU-KEG/MAVEN-dataset/tree/main/baselines/DMBERT), which implements DMBERT in a relatively modern and cleaner way.
